tests/src/JIT/Directed/pinvoke/user32menu.def

```diff
@@ -0,0 +1,5 @@
+EXPORTS
```

Comment:
This file isn't needed since `EXPORT_API  is defined in source.

Reply:
Fair enough.  I'm used to `.def` files being preferred, and I just followed the example of a nearby project, but I see that there are only a handful of `.def` files in the test tree, so best not to add more.